chaos_genius/views/anomaly_data_view.py

```diff
@@ -390,6 +390,47 @@ def kpi_anomaly_retraining(kpi_id: int):
     return jsonify({"msg": f"retraining started for KPI: {kpi_id}"})
 
 
+@blueprint.route("/<int:kpi_id>/disable-anomaly", methods=["GET", "POST"])
+def disable_anomaly(kpi_id): 
+    """API end point which disables analytics by modifying the run_anomaly flag."""
+    kpi = cast(Optional[Kpi], Kpi.get_by_id(kpi_id))
+    if kpi is not None:
+        # check if anomaly is setup
+        if kpi.run_anomaly and kpi.anomaly_params:
+            kpi.run_anomaly = False
+            kpi.update(commit=True)
+            message = f"Disabled Analytics for KPI ID: {kpi_id}"
+            status = "success"
+        else:
+            message = f"Falied to Disable Analytics for KPI ID: {kpi_id}"
+            status = "failure"
+    else:
+        message = f"KPI {kpi_id} could not be retreived."
+        status = "failure"
+    logger.info(message)
+    return jsonify({"msg": message, "status": status})
+
+
+@blueprint.route("/<int:kpi_id>/enable-anomaly", methods=["GET", "POST"])
+def enable_anomaly(kpi_id):
+    """API end point which enables analytics by modifying the run_anomaly flag."""
+    kpi = cast(Optional[Kpi], Kpi.get_by_id(kpi_id))
+    if kpi is not None:
+        if (not kpi.run_anomaly) and kpi.anomaly_params:
+            kpi.run_anomaly = True
+            kpi.update(commit=True)
+            message = f"Enabled Analytics for KPI ID: {kpi_id}"
+            status = "success"
+        else:
+            message = f"Falied to Enable Analytics for KPI ID: {kpi_id}"
```

Comment:
Same here
```suggestion
            message = f"Failed to Enable Anomaly because it is either already enabled or not set up for KPI ID: {kpi_id}"
```